pkgs/by-name/pa/paperlib-bin/package.nix

```
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchurl
+, appimageTools
+, undmg
+,
+}:
+let
+  pname = "paperlib";
+  version = "3.1.6";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+          url = "https://github.com/Future-Scholars/${pname}/releases/download/release-electron-${version}/Paperlib_${version}.dmg";
+          hash = "sha256-d9vEFx59K15PO7DJYJQ2fjiagqa8oJLtoawILDF9IKc=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/Future-Scholars/${pname}/releases/download/release-electron-${version}/Paperlib_${version}.AppImage";
+          hash = "sha256-2xbn9UWlcf37n9jZdZKyyevzsag6SW9YuQH/bYCRmLQ=";
+        };
+      }.${stdenv.system} or (throw "Unsupported system: ${stdenv.system}");
+
+  passthru = {
+    inherit pname version src;
+  };
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
remove `with lib;` and prepend `lib.` where required